db/seeds.rb

```diff
@@ -1,9 +1,162 @@
+end
+
+# Create degrees
+degrees = [
+  "Doctor of Medicine (MD)",
+  "Bachelor of Medicine, Bachelor of Surgery (MBBS)",
+  "Master of Surgery (MS)",
+  "Diplomate of National Board (DNB)",
+  "Bachelor of Ayurvedic Medicine and Surgery (BAMS)",
+  "Bachelor of Homoeopathic Medicine and Surgery (BHMS)",
+  "Bachelor of Science in Nursing (B.Sc Nursing)",
+  "Master of Science in Nursing (M.Sc Nursing)",
+  "Doctor of Osteopathic Medicine (DO)",
+  "Bachelor of Physiotherapy (BPT)"
+].map do |degree|
+  Degree.find_or_create_by!(name: degree)
+end
+
+# Create institutes
+institutes = [
+  "All India Institute of Medical Sciences (AIIMS)",
+  "Christian Medical College (CMC), Vellore",
+  "Armed Forces Medical College (AFMC), Pune",
+  "Post Graduate Institute of Medical Education and Research (PGIMER), Chandigarh",
+  "Jawaharlal Institute of Postgraduate Medical Education and Research (JIPMER), Puducherry",
+  "King George's Medical University (KGMU), Lucknow",
+  "Maulana Azad Medical College, Delhi",
+  "Manipal University, Karnataka",
+  "Kasturba Medical College, Manipal",
+  "Banaras Hindu University (BHU), Varanasi",
+  "Government Medical College, Mumbai",
+  "Sanjay Gandhi Postgraduate Institute of Medical Sciences, Lucknow",
+  "Lady Hardinge Medical College, Delhi",
+  "Sri Ramachandra Institute of Higher Education and Research, Chennai",
+  "Gujarat Cancer Research Institute, Ahmedabad",
+  "Narayana Health, Bangalore",
+  "Indira Gandhi Institute of Medical Sciences, Patna",
+  "B.J. Medical College, Ahmedabad",
+  "Bhopal Memorial Hospital and Research Centre, Bhopal",
+  "National Institute of Mental Health and Neuro Sciences (NIMHANS), Bangalore"
+].map do |institute|
+  Institute.find_or_create_by!(name: institute)
+end
+
+# Create registration councils
+registration_councils = [
+  "Medical Council of India (MCI)",
+  "Delhi Medical Council",
+  "Maharashtra Medical Council",
+  "Karnataka Medical Council",
+  "Tamil Nadu Medical Council",
+  "Gujarat Medical Council",
+  "Uttar Pradesh Medical Council",
+  "Andhra Pradesh Medical Council",
+  "Kerala Medical Council",
+  "West Bengal Medical Council"
+].map do |council|
+  RegistrationCouncil.find_or_create_by!(name: council)
+end
+
+# Create address for establishments and patients
+address = Address.create!(
+  address_line_1: Faker::Address.street_address,
+  address_line_2: Faker::Address.secondary_address,
+  city: Faker::Address.city,
+  state: Faker::Address.state,
+  country: "India",
+  pin_code: Faker::Address.zip_code
+)
+
+# Create establishments and associate with address
+establishment = Establishment.create!(
+  name: Faker::Company.name,
+  address_id: address.id,   # associate with address using address_id
+  latitude: Faker::Address.latitude,
+  longitude: Faker::Address.longitude,
+  maps_location: "https://goo.gl/maps/#{Faker::Address.street_address.gsub(' ', '-')}"  # Example for a map location
+)
+
+# Create doctor profiles
```

Comment:
Couple of suggestions for fake data.

This code creates only 1 doctor_profile, it should be atleast 20.
This code creates only 1 patient_profile, it should be atleast 50.

This code adds only 1 review it should be atleast 20-50
This code adds only 1 service it should be atleast 20-50

Reply:
I think done also, 


Ignore my comment on the code I will take them off soon. 😂